### DISCUSSION TITLE
Welcome to orka-desktop Discussions!

### DISCUSSION THREAD

**Discussion by jasoncdavis0** [macstadium/orka-desktop#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with Orka Desktop!

**Comment by wddwycc** [comment]
Hi team, great work for the Orka desktop client, congrates to the release! 
Do we have a way to transfer files in between local and VM currently ?

**Reply by abojak** [reply]
Hi Duan thanks for the feedback! Currently Orka Desktop does not allow data transfer operations between Guest & Host. The workaround is to transfer via a cloud app (social media, mail, cloud storage etc.) by transferring data within the cloud app by logging into the app in the host and/or guest, this includes copy/paste operations between guest and host. Transferring data between host<->guest within the Orka Desktop application is in the roadmap release date pending.
